### PR TITLE
Add Breeze auth services

### PR DESCRIPTION
**Pull request by rick-astral-cat**
This commit add required packages to start developing frontend and authentication services.
This packages are loaded with npm and composer so run npm run build and composer update.
-Livewire
-Laravel Breeze
-Material Icons

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,10 +10,12 @@
         "guzzlehttp/guzzle": "^7.2",
         "laravel/framework": "^10.10",
         "laravel/sanctum": "^3.2",
-        "laravel/tinker": "^2.8"
+        "laravel/tinker": "^2.8",
+        "livewire/livewire": "^3.0"
```
Comment (crazybolillo) [inline composer.json]: Is this required for authorization?

File: package.json [hunk 2]
```diff
@@ -6,8 +6,20 @@
...
+        "postcss": "^8.4.6",
+        "postcss-scss": "^4.0.7",
+        "tailwindcss": "^3.1.0",
         "vite": "^4.0.0"
+    },
+    "dependencies": {
+        "bootstrap": "^5.3.1",
+        "jquery": "^3.7.0",
+        "material-icons": "^1.13.10",
+        "postcss-loader": "^7.3.3"
```
Comment (crazybolillo) [inline package.json]: I don't think these are added by breeze, at least from what I  tested. Maybe by livewire? But I think we should do this step by step.

File: resources/css/app.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@import "material-icons";
+@import "bootstrap";
```
Comment (crazybolillo) [inline resources/css/app.css]: Will you try to get rid of bootstrap? From what I see at least the login page is already built, I don't think we need extra styling, so I don't see the need for bootstrap as of now :thinking:

**Review by crazybolillo — COMMENTED** [review]
It is hard to tell what change comes from installing breeze and what changes are yours. I suggest splitting it into two commits:

- First commit with everything after running `php artisan breeze:install`
- Second commit with your modifications

Also, the goal should be clear. In this case stuff like livewire or bootstrap are added but not really used. I think this PR should only focus on making user authentication work, so any extra dependencies like bootstrap or jqueyr should not be here, it is better to add them once you add the components that need them.

**Comment by crazybolillo** [comment]
Looks good overall, just need some organization. It will make things easier if we ever want to go back and review old code.

File: composer.json (hunk 1 above)
Comment (rick-astral-cat) [reply composer.json]: Laravel tinker is a command line to interact with Laravel ORM installed by Laravel Breeze. I think this utility could be helpful sometimes. 
Livewire is neccesary to react components, but we can not added yet on this PR.

**Comment by crazybolillo** [comment]
@rick-astral-cat

I have rebased it to fix the CI. If you wish you can merge this or push your commits to customize the login page here.

**Pull request merged**